docs/source/en/model_doc/mot.md

```diff
@@ -18,19 +18,31 @@ rendered properly in your Markdown viewer.
 
 ## Overview
 
-The MixtureOfTokens model was proposed in [<INSERT PAPER NAME HERE>](<INSERT PAPER LINK HERE>) by <INSERT AUTHORS HERE>.
-<INSERT SHORT SUMMARY HERE>
+The MixtureOfTokens model was proposed in [MIXTURE OF TOKENS: EFFICIENT LLMS THROUGH
+CROSS-EXAMPLE AGGREGATION](https://arxiv.org/pdf/2310.15961.pdf) by Szymon Antoniak, Sebastian Jaszczur, Michał Krutul, Maciej Pióro, Jakub Krajewski, Jan Ludziejewski, Tomasz Odrzygóźdź, Marek Cygan.
```

Comment:
How about link pointing to arxiv instead of actual pdf?

```suggestion
CROSS-EXAMPLE AGGREGATION](https://arxiv.org/abs/2310.15961) by Szymon Antoniak, Sebastian Jaszczur, Michał Krutul, Maciej Pióro, Jakub Krajewski, Jan Ludziejewski, Tomasz Odrzygóźdź, Marek Cygan.
```